### DISCUSSION TITLE
Sorting issue

### DISCUSSION THREAD

**Discussion by kaisiant** [Automattic/mongoose#10798] [Q&A] [answered]
Hi friends, just want to know can I sort the parent based on the child value at this moment?
What I want to do is same as Issue [6736](https://github.com/Automattic/mongoose/issues/6736)
![image](https://user-images.githubusercontent.com/26655111/135066519-f5bc24a5-1856-4e29-9b0b-2f04c316fd41.png)

**Comment by vkarpov15** [comment, marked as answer]
@kaisian97 no, you can't. If you want to sort parent by child properties, you should denormalize and put the child property in the parent. [Store what you query for.](https://thecodebarbarian.com/mongoose-schema-design-pattern-store-what-you-query-for.html)

**Reply by kaisiant** [reply]
Thanks for the link, it helps me a lot.
